components/Tabs2/src/private/MDCTabBarViewItemView.m

```diff
@@ -23,8 +23,82 @@
 /** The minimum height of any item view. */
 static const CGFloat kMinHeight = 48;
 
+/// Outer edge padding from spec: https://material.io/go/design-tabs#spec.
+static const UIEdgeInsets kEdgeInsets = {.top = 12, .right = 16, .bottom = 12, .left = 16};
+
+/// File name of the bundle (without the '.bundle' extension) containing resources.
+static NSString *const kResourceBundleName = @"MaterialTabs";
+
+/// String table name containing localized strings.
+static NSString *const kStringTableName = @"MaterialTabs";
+
+/// Scale factor applied to the title of bottom navigation items when selected.
```

Comment:
This looks like it was copied from MDCTabBar - please don't copy the code from the existing Tabs.  Bottom Navigation is a separate component and is not part of Tabs.

Reply:
Done.